tests/utils.go

```diff
@@ -601,7 +601,10 @@ func AfterTestSuitCleanup() {
 	if DeployTestingInfrastructureFlag {
 		WipeTestingInfrastructure()
 	}
-	removeNamespaces()
+	err := removeNamespaces()
```

Comment:
i dont think kubevirtci bump should be mixed with other code please ?
i would split PRs if possible

Reply:
Done